frontend/javascripts/oxalis/view/right-menu/dataset_info_tab_view.js

```diff
@@ -192,12 +192,22 @@ class DatasetInfoTabView extends React.PureComponent<Props> {
       description: datasetDescription,
       owningOrganization,
     } = this.props.dataset;
+    const getEditSettingsIcon = () => (
+      <Tooltip title="Edit dataset settings">
+        <Button
+          type="text"
+          icon={<EditOutlined />}
+          href={`/datasets/${owningOrganization}/${datasetName}/edit`}
```

Comment:
Didn't test this, but this should ensure that the link is always opened in another tab.

```suggestion
          href={`/datasets/${owningOrganization}/${datasetName}/edit`}
          target="_blank"
```

Reply:
:+1: Nice, I didn't think of this!